flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
```

Comment:
`animationList` is a noun, flixel functions should be either `verb` or `verbNoun`, also, `animation.animationList` is redundant. This should be `getList`.

This also needs documentation. Above the function, there should should be a description of that this does, like so: 
```hx
/**
 * blah blah
 */
```

Reply:
I think it's now resolved